addons/goggles/functions/fnc_handleFired.sqf

```diff
@@ -24,17 +24,15 @@ if (rain > 0.1) exitWith {true};
 // effect only aplies when lying on the ground
 if (stance _unit != "PRONE") exitWith {true};
 
-private ["_position", "_particleConfig", "_cloudType", "_surface", "_bullets"];
-
 // check if the unit really is on the ground and not in a building
-_position = getPosATL _unit;
+private _position = getPosATL _unit;
 
 if (_position select 2 > 0.2) exitWith {true};
 
 // get weapon dust effect
-_particleConfig = configFile >> "CfgWeapons" >> _weapon >> "GunParticles";
+private _particleConfig = configFile >> "CfgWeapons" >> _weapon >> "GunParticles";
 
-_cloudType = "";
+private private _cloudType = "";
```

Comment:
double private

Reply:
If I got a double private there it means I forgot a private below.. Sometimes I would click a pixel to far left and hit the line number instead of the text... Atom is somewhat crap I guess.

Reply:
Jup. _surface on line 51 was missing